src/translations/screens/subscreens/PurchaseOverview.ts

```diff
@@ -98,10 +108,14 @@ const PurchaseOverviewTexts = {
   summary: {
     price: (priceString: string) =>
       _(`Total ${priceString} kr`, `Total ${priceString} kr`),
-    message: _(
+    messageInZone: _(
       `Gjelder for buss/trikk i valgte soner`,
       `Applies for bus/tram in selected zones`,
     ),
+    messageInArea: _(
+      `Gjelder for buss/trikk i valgt område`,
+      `Applies for bus/tram in selected area`,
+    ),
```

Comment:
Little unsure about this. We can't say that always, and for all transport companies, that when `zoneSelectionMode: 'none'` the selected area will be described in the product description.

Reply:
For this case is when `zoneSelectionMode` is not `none`

Reply:
Yes, and that's the problem. The text mentions "in selected area", but it is no zone selection for selecting the area. This works for AtB night tickets, but there is no guarantee that all products with `zoneSelectionMode: none` will describe the area in the product description. Remember, the app will also sell tickets for NFK, Fram and probably ATK.